test/e2e/e2e.go

```diff
@@ -0,0 +1,14 @@
+// Copyright © 2018 The Kubernetes Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
```

Comment:
what is this file? could you delete it?

Reply:
This suppress build errors when using an editor. It complains that the package has a test file, but nothing else.